src/plugins/telemetry/telemetry_impl.h

```diff
@@ -120,19 +120,19 @@ class TelemetryImpl : public PluginImplBase {
     void set_rc_status(bool available, float signal_strength_percent);
     void set_unix_epoch_time_us(uint64_t time_us);
 
-    void process_position_velocity_ned(const mavlink_message_t &message);
-    void process_global_position_int(const mavlink_message_t &message);
-    void process_home_position(const mavlink_message_t &message);
-    void process_attitude_quaternion(const mavlink_message_t &message);
-    void process_mount_orientation(const mavlink_message_t &message);
-    void process_imu_reading_ned(const mavlink_message_t &message);
-    void process_gps_raw_int(const mavlink_message_t &message);
-    void process_extended_sys_state(const mavlink_message_t &message);
-    void process_sys_status(const mavlink_message_t &message);
-    void process_heartbeat(const mavlink_message_t &message);
-    void process_statustext(const mavlink_message_t &message);
-    void process_rc_channels(const mavlink_message_t &message);
-    void process_unix_epoch_time(const mavlink_message_t &message);
+    void process_position_velocity_ned(const mavlink_message_t& message);
+    void process_global_position_int(const mavlink_message_t& message);
+    void process_home_position(const mavlink_message_t& message);
+    void process_attitude_quaternion(const mavlink_message_t& message);
+    void process_mount_orientation(const mavlink_message_t& message);
+    void process_imu_reading_ned(const mavlink_message_t& message);
+    void process_gps_raw_int(const mavlink_message_t& message);
+    void process_extended_sys_state(const mavlink_message_t& message);
+    void process_sys_status(const mavlink_message_t& message);
+    void process_heartbeat(const mavlink_message_t& message);
+    void process_statustext(const mavlink_message_t& message);
+    void process_rc_channels(const mavlink_message_t& message);
+    void process_unix_epoch_time(const mavlink_message_t& message);
```

Comment:
This is still wrong.

Reply:
I'm sorry, I can't see it... what is wrong?

Reply:
I'm sorry, I can't see it... what is wrong?

Reply:
Sorry, I must have looked at a wrong diff. This is fine.